packages/contracts/src/networks.ts

```diff
@@ -722,43 +759,26 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     decimals: 18,
     queryFilterBlockLimit: 2000,
     legacyTx: false,
+    actionGasLimitBuffer: false,
+  },
+  {
+    name: 'rootstock',
+    chainId: BigInt(30),
+    rpcUrl: () => process.env.ROOTSTOCK_RPC_URL!,
+    etherscan: {
+      apiURL: 'https://rootstock.blockscout.com/api',
+      browserURL: 'https://rootstock.blockscout.com/',
+      envKey: 'ROOTSTOCK_ETHERSCAN_API_KEY',
+      blockExplorer: 'Blockscout',
+    },
+    currency: 'RBTC',
+    dripSize: '0.001',
+    requiredEnvVariables: ['ROOTSTOCK_RPC_URL'],
+    dripVersion: 2,
+    networkType: 'Mainnet',
+    legacyTx: true,
+    decimals: 8,
+    queryFilterBlockLimit: 2000,
```

Comment:
Another general question: does the `queryFilterBlockLimit` apply to all networks, even if we're using a paid RPC url? I'm surprised that node providers on Ethereum restrict this to 2000 blocks

Reply:
There are a few different networks where this is an issue, but it isn't an issue on all of them. However, previously I just had a hardcoded limit to 2k on all networks. So I just set this to 2k for all networks to maintain the current behavior.